0.data-download/README.md

```diff
@@ -0,0 +1,30 @@
+### Description of the Data ###
+
+# DepMap Public 22Q4
+
+This DepMap release contains data from in vitro studies of genetic dependencies in cancer cell lines using CRISPR/Cas9 loss-of-function screens from project Achilles, as well as genomic characterization data from the CCLE project.
+
+Data resource:
+https://depmap.org/portal/download/all/
+
+
+
+
```

Comment:
```suggestion
```

It is safe to delete excess new lines. We typically like to keep `.md` files clean and consistently formatted.